SasquatchMac/SasquatchMac/ViewControllers/TransmissionViewController.swift

```diff
@@ -226,8 +226,8 @@ class TransmissionViewController: NSViewController, NSTableViewDataSource, NSTab
       guard let identifier = tableColumn?.identifier else {
         return nil
       }
-      let view = tableView.make(withIdentifier: identifier, owner: nil)
-      if (identifier == "value") {
+        let view = tableView.makeView(withIdentifier: identifier, owner: nil)
```

Comment:
Please revisit formatting in all places, not only in line that has my comment

Reply:
ok，i will do that.